src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultBraceSmartIndenterFactory.cs

```diff
@@ -50,9 +50,7 @@ public override BraceSmartIndenter Create(VisualStudioDocumentTracker documentTr
 
             _joinableTaskContext.AssertUIThread();
 
-            var braceSmartIndenter = new BraceSmartIndenter(_joinableTaskContext, documentTracker, _codeDocumentProvider, _editorOperationsFactory);
-
-            return braceSmartIndenter;
+            return new BraceSmartIndenter(_joinableTaskContext, documentTracker, _codeDocumentProvider, _editorOperationsFactory);
```

Comment:
This pattern of not directly returning things, but assigning them to a local variable first, is used heavily throughout the Razor codebase. "Cleaning" these up is definitely something that should be discussed in an issue first.

Reply:
Uh, I thought that there used to be some actions with this local and then they became unnecessary, totally didn't expect it as an explicit style choice

Reply:
It was a deliberate style choice by previous Razor code owners to make debugging easier. I don't know that it's a style choice anyone on the current team would necessarily make, or enforce in new code, but its not something we've talked about and decided to go back and address, either.

Reply:
Thanks for bringing this up @davidwengier. This is something I've noticed too and have been tempted to clean up. :smile: I'll create an issue to discuss code style clean up throughout the repo.

Reply:
Created #6941